frontend/src/Helpers/Props/icons.js

```diff
@@ -162,6 +162,7 @@ export const KEYBOARD = farKeyboard;
 export const LOGOUT = fasSignOutAlt;
 export const MANAGE = fasListCheck;
 export const MEDIA_INFO = farFileInvoice;
+export const MENU = fasBars;
```

Comment:
This icon is already used for the collapsing sidebar on mobile, so I don't think we want to use it here.

Reply:
I've reverted the Icon change